src/components/search/Search.tsx

```diff
@@ -0,0 +1,88 @@
+import {Text, TextInput, View, FlatList} from 'react-native';
+import {useEffect, useState} from 'react';
+import Icon from 'react-native-vector-icons/Feather';
+
+import {theme} from '@theme/themes';
+import {fetchData} from '@network/apiMethods';
+import MovieCard from '@components/movieCard';
+import {useLoadingContext} from '@contexts/LoadingContext';
+import Loader from '@components/loader';
+
+import {styles} from './styles';
+
+const Search = () => {
+  const {isLoading, setIsLoading} = useLoadingContext();
+  const [searchTerm, setSearchTerm] = useState('');
+  const [searchedData, setsearchedData] = useState([]);
+
+  const getData = async () => {
+    setIsLoading(true);
+
+    const searchedMovies = await fetchData(`/search/movie`, {
+      params: {query: searchTerm, page: '1'},
+    });
+
+    setsearchedData(
+      searchedMovies?.results?.map(
```

Comment:
extract to a variable

make sure you remember the comments from PRs, I have already mentioned this in a previous one